website/language.py

```diff
@@ -146,7 +146,7 @@
     u'This {category} contains links to other projects. These links will be '
     u'copied into your registration, but the projects that they link to will '
     u'not be registered. If you wish to register the linked projects, they '
-    u'must be registered separately. Learn more about <a href="http://help.osf.io'
+    u'must be registered separately. Learn more about <a href="https://help.osf.io/article/199-link-to-a-project'
     u'/m/links_forks/l/524112-link-to-a-project">links</a>.'
```

Comment:
The formatting of this block is a bit odd, but this is the line that should receive `/article/199-link-to-a-project`, replacing `/m/links_forks/l/524112-link-to-a-project`